experimenter/experimenter/experiments/models.py

```diff
@@ -1137,6 +1137,12 @@ def get_firefox_min_version_display(self):
     def get_firefox_max_version_display(self):
         return self.firefox_max_version.replace("!", "0")
 
+    @property
+    def kinto_collection(self):
+        # Note: this can throw if there are conflicting features targeting
+        # different collections.
+        return self.application_config.get_kinto_collection_for(self)
```

Comment:
What are the conditions we'd throw?  Shouldn't it promote to secure collection if any feature needs it and just leave it there?